internal/quickstart/choose_sdk.go

```diff
@@ -96,48 +96,46 @@ func (m chooseSDKModel) View() string {
 }
 
 type sdkDetail struct {
-	canonicalName   string
-	displayName     string
-	index           int
-	kind            string
-	url             string // custom URL if it differs from the other SDKs
-	hasInstructions bool   // to remove when we get all instructions loaded
+	canonicalName string
+	displayName   string
+	index         int
+	kind          string
+	url           string // custom URL if it differs from the other SDKs
```

Comment:
keeping this in here for now because I think I want to pull these out of the instruction files